src/WebJobs.Script/Utility.cs

```diff
@@ -127,12 +128,27 @@ internal static async Task DelayAsync(int timeoutSeconds, int pollingIntervalMil
 
         internal static TimeSpan ComputeBackoff(int exponent, TimeSpan? unit = null, TimeSpan? min = null, TimeSpan? max = null)
         {
+            TimeSpan maxValue = max ?? TimeSpan.MaxValue;
```

Comment:
We have the same code/issue in V1 and need a fix there as well: https://github.com/Azure/azure-functions-host/blob/8fc8f9f1bb35ac7e4a20265957986e9ebb86caaa/src/WebJobs.Script/Utility.cs#L86

Reply:
And also in the core SDK for RandomizedExponentialBackoffStrategy, though in that case a double is used, and it's not clear that we've seen actual issues here: https://github.com/Azure/azure-webjobs-sdk/blob/2cdc3d44c572a10e71eb6ff8d63249117510dd9b/src/Microsoft.Azure.WebJobs.Host/Timers/RandomizedExponentialBackoffStrategy.cs#L68

Reply:
I wonder if in most use-cases for this we just never retry this many times? We can potentially retry forever in this case, so it seems to be the most problematic.